### DISCUSSION TITLE
Youdao

### DISCUSSION THREAD

**Discussion by iG8R** [pot-app/pot-desktop#714] [Q&A]
Hello.
Is there any way to get the translation and dictionary entry without any app_ID or key like I do it on https://www.youdao.com/result?word=全&lang=en, just entering words or texts?

![image](https://github.com/pot-app/pot-desktop/assets/11407417/d8ce1e6f-6455-43ae-9f12-797898e0f43b)

PS. I'm a foreigner, so getting the app_ID and key is an almost impossible challenge due to the mandatory requirement of having a Chinese phone number.

E.g. how it implemented here https://github.com/tisfeng/Raycast-Easydict
![image](https://raw.githubusercontent.com/tisfeng/ImageBed/main/uPic/iShot_2023-03-17_18.01.22_11zon-1679056100.jpg)

**Comment by Pylogmon** [comment]
Check out the pot plugins list

**Comment by iG8R** [comment]
Thanks a lot for the advice, I tried it, but this plugin (https://github.com/TechDecryptor/pot-app-translate-plugin-youdao/releases/tag/1.4.0) doesn't exactly output what appears on the original yuodao page in response to the same request.

![image](https://github.com/pot-app/pot-desktop/assets/11407417/a7c4db33-391e-4539-b40d-188cf8a96943)
